queue_services/account-mailer/q_cli.py

```diff
@@ -0,0 +1,129 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+# Copyright © 2019 Province of British Columbia
+#
+# Licensed under the Apache License, Version 2.0 (the 'License');
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an 'AS IS' BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Service for listening and handling Queue Messages.
+
+This service registers interest in listening to a Queue and processing received messages.
+"""
+import asyncio
+import functools
+import getopt
+import json
+import os
+import random
+import signal
+import sys
+
+from nats.aio.client import Client as NATS  # noqa N814; by convention the name is NATS
+from stan.aio.client import Client as STAN  # noqa N814; by convention the name is STAN
+
+from entity_queue_common.service_utils import error_cb, logger, signal_handler
+
+
+async def run(loop, old_identifier, new_identifier):  # pylint: disable=too-many-locals
+    """Run the main application loop for the service.
+
+    This runs the main top level service functions for working with the Queue.
+    """
+    # NATS client connections
+    nc = NATS()
+    sc = STAN()
+
+    async def close():
+        """Close the stream and nats connections."""
+        await sc.close()
+        await nc.close()
+
+    # Connection and Queue configuration.
+    def nats_connection_options():
+        return {
+            'servers': os.getenv('NATS_SERVERS', 'nats://127.0.0.1:4222').split(','),
+            'io_loop': loop,
+            'error_cb': error_cb,
+            'name': os.getenv('NATS_CLIENT_NAME', 'entity.filing.tester')
```

Comment:
Can we put the defaults as we discussed, starting with account.*. This is a TODO for pay q listener as well.

Reply:
i will do it in next PR